Purchases/Support/BeginRefundRequestHelper.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  BeginRefundRequestHelper.swift
+//
+//  Created by Madeline Beyl on 10/13/21.
+
+import Foundation
+
+/**
+ * Helper class responsible for handling any non-store-specific code involved in beginning a refund request.
+ * Delegates store-specific operations to `SK2BeginRefundRequestHelper`.
+ */
+class BeginRefundRequestHelper {
+
+    private let systemInfo: SystemInfo
+
+#if os(iOS) || targetEnvironment(macCatalyst)
+    @available(iOS 15.0, *)
+    @available(watchOS, unavailable)
+    @available(tvOS, unavailable)
+    lazy var sk2Helper = SK2BeginRefundRequestHelper()
```

Comment:
Ditto: `private(set)`

Reply:
Oh in that case I wouldn't bother. Unfortunately there's no "private for tests" so disregard.